chcemvediet/apps/anonymization/admin.py

```diff
@@ -72,3 +72,66 @@ def get_urls(self):
                 url(r'^(.+)/download/$', download_view, name=u'{}_{}_download'.format(*info)),
                 )
         return urls + super(AttachmentNormalizationAdmin, self).get_urls()
+
+@admin.register(AttachmentRecognition, site=admin.site)
+class AttachmentRecognitionAdmin(admin.ModelAdmin):
+    date_hierarchy = u'created'
+    list_display = [
+            u'id',
+            decorate(
+                lambda o: admin_obj_format(o.attachment, u'{obj}'),
+                short_description=u'Attachment',
+                admin_order_field=u'attachment',
+            ),
+            u'successful',
+            decorate(
+                lambda o: admin_obj_format(o, u'{obj.file.name}', link=u'download'),
+                short_description=u'File',
+                admin_order_field=u'file',
+                ),
+            u'name',
+            u'content_type',
+            u'created',
+            decorate(
+                lambda o: filesize(o.size),
+                short_description=u'Size',
+                admin_order_field=u'size',
+                ),
+            ]
+    list_filter = [
+            u'created',
+            u'content_type',
+            u'successful',
+            ]
+    search_fields = [
+            u'=id',
+            u'file',
+            u'name',
+            u'content_type',
+            u'debug',
+            ]
+    ordering = [
+            u'-id',
+            ]
+    exclude = [
+            u'file',
+            ]
+    readonly_fields = [
+            ]
+    raw_id_fields = [
+            ]
+    inlines = [
+            ]
+
+    def download_view(self, request, attachment_normalization_pk):
+        attachment_recognition = AttachmentRecognition.objects.get_or_404(
+            pk=attachment_normalization_pk)
+        return download(request, attachment_recognition)
```

Comment:
Hm. Necakal som, ze ten isty `download` view sa bude dat pouzit aj pre `AttachmentRecognition` aj pre `AttachmentNormalization`. Ale zjavne funguje. Potom vsak by rovnako mal fungovat `download` view z `poleno/attachments/views.py` a nemusime robit separatny download view tu. Pls otestuj, ci funguje ten vseobecny download view z `poleno/attachments/views.py` a ak ano, tak ho pouzi a nas download view specificky pre `attachment_normalization` zmaz.

V tom pripade dopln k `download` v `poleno/attachments/views.py` dosctring nieco na styl "Download view for attachments and attachment like objects"

Reply:
:ok: